apps/evalite-ui/app/routes/eval.$name.lazy.tsx

```diff
@@ -0,0 +1,377 @@
+import { average } from "@evalite/core/utils";
+import {
+  createLazyFileRoute,
+  Link,
+  Outlet,
+  useMatches,
+} from "@tanstack/react-router";
+import { XCircleIcon } from "lucide-react";
+import type * as React from "react";
+import { useContext } from "react";
+import { DisplayInput } from "~/components/display-input";
+import { InnerPageLayout } from "~/components/page-layout";
+import { getScoreState, Score } from "~/components/score";
+import { MyLineChart } from "~/components/ui/line-chart";
+import { LiveDate } from "~/components/ui/live-date";
+import { Separator } from "~/components/ui/separator";
+import {
+  Table,
+  TableBody,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "~/components/ui/table";
+import { cn } from "~/lib/utils";
+import { TestServerStateContext } from "~/use-subscribe-to-socket";
+import { formatTime, isArrayOfRenderedColumns } from "~/utils";
+
+export const Route = createLazyFileRoute("/eval/$name")({
```

Comment:
Files were big enough I went for codesplitting. See https://tanstack.com/router/latest/docs/framework/react/guide/code-splitting#using-the-lazytsx-suffix for more details. bascially you have to separate components from loaders and other route info.

Reply:
lmao I had the `automaticCodeSplitting` option set to true from the tutorial so did a revert